Pinta.Core/Effects/GradientRenderer.cs

```diff
@@ -17,75 +17,56 @@ public abstract class GradientRenderer
 	private readonly BinaryPixelOp normal_blend_op;
 	private ColorBgra start_color;
 	private ColorBgra end_color;
-	private PointD start_point;
-	private PointD end_point;
-	private bool alpha_blending;
-	private bool alpha_only;
-
 	private bool lerp_cache_is_valid = false;
 	private readonly byte[] lerp_alphas;
 	private readonly ColorBgra[] lerp_colors;
 
+	protected internal GradientRenderer (
+	bool alphaOnly,
```

Comment:
The formatting looks weird here, unless the GitHub diff is behaving strangely - the parameters should be indented over more

Reply:
@cameronwhite the formatting did indeed turn out weird by accident. I've now fixed it